src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyCommentBeforeLeftCurly2.java

```diff
@@ -25,29 +30,29 @@ public class InputLeftCurlyCommentBeforeLeftCurly2 {
     private long[] countList;
 
     void method1() /* comment
-    comment */ { // ok until #11410
+    comment */ { // ok
     }
 
     void method2() /********
-    comment */ {  // ok
+    comment */ { // ok
         if (!Arrays.equals(this.countList, countList)
-            /* || !Arrays.equals(this.whereCreated, other.whereCreated) */ ) { // ok
+            /* || !Arrays.equals(this.whereCreated, other.whereCreated) */ ) {
         }
     }
 
     void method3() /*****
-    ********/ { } // ok until #11410
+    ********/ { } // ok
 
 
-    InputLeftCurlyCommentBeforeLeftCurly2() /**************/ { } // ok
+    InputLeftCurlyCommentBeforeLeftCurly2() /**************/ { }
```

Comment:
why it is ok?

Reply:
Because Left curly is on the same line as expectation 
i have rewrite ok their